features/design_principles.feature

```diff
@@ -11,19 +11,6 @@ Feature: Design Principles
     And I should see "Start with user needs"
 
   @normal
-  Scenario: check Service Manual
-    When I visit "/service-manual"
-    Then I should get a 200 status code
-    And I should see "Government Service Design Manual"
-
-  @normal
-  Scenario: check Service Manual search
```

Comment:
It seems redundant to me – search is independently tested and we're just using that with a filter_manual parameter.